site/content/docs/main/api-types/schedule.md

```diff
@@ -94,6 +94,13 @@ spec:
       matchLabels:
         app: velero
         component: server
+    # Individual object when matched with any of the label selector specified in the set are to be included in the backup. Optional.
+    # orLabelSelectors as well as labelSelector cannot co-exist, only one of them can be specified in the backup request
+    orLabelSelectors:
```

Comment:
I think it will be good to make the same change in older versions doc as well - at least in 1.11.